models/generated/workloads/reservation.go

```diff
@@ -73,6 +73,7 @@ type ReservationData struct {
 	PublicIPs               []PublicIP            `bson:"public_ips" json:"public_ips"`
 	ExpirationProvisioning  schema.Date           `bson:"expiration_provisioning" json:"expiration_provisioning"`
 	ExpirationReservation   schema.Date           `bson:"expiration_reservation" json:"expiration_reservation"`
+	VirtualMachines         []VirtualMachine      `bson:"virtual_machine" json:"virtual_machine"`
```

Comment:
Ahm, also ReservationData is deprecated and it's kept only for backward compatibility since the database still has some data from this structure. I would also remove the changes here since it's not maintained and might confuse people in the future (hopefully not long before the explorer is obsolete)